instrumentation/kamon-pekko/src/main/resources/reference.conf

```diff
@@ -153,7 +153,6 @@ kanela.modules {
       "kamon.instrumentation.pekko.instrumentations.EventStreamInstrumentation",
       "kamon.instrumentation.pekko.instrumentations.ActorRefInstrumentation",
       "kamon.instrumentation.pekko.instrumentations.DispatcherInstrumentation",
-      "kamon.instrumentation.pekko.instrumentations.ActorMonitorInstrumentation",
```

Comment:
This doesn't seem to fix any tests, and breaks things with scala 3

Reply:
Actually I've figured this one out. Accepting the argument to the advice as an Envelope causes the scala 3 compiler to do some sort of magic to strip the HasContext mixin out. Mysterious, but resolvable.